.github/workflows/build-release-packages.yml

```diff
@@ -23,7 +23,7 @@ jobs:
   # This job provides the version metadata from the tag for the other jobs to use.
   release-build-get-meta:
     name: Get metadata to build
-    if: github.repository == 'valkey-io/valkey'
+    if: github.event_name == 'workflow_dispatch' || github.repository == 'valkey-io/valkey'
```

Comment:
Is this to be able to test the workflow in a fork?

Reply:
Yes, contributors can use `workflow_dispatch` in their fork to trigger the workflow

Reply:
![Screenshot 2024-12-19 at 12 24 53 PM](https://github.com/user-attachments/assets/b80dac59-b591-4a9a-a67e-ecdd8f63ce2b)

Reply:
Thats what you wrote in the top comment, yey...